test/functional/sql-tag/sql-tag-parameters-postgres.ts

```diff
@@ -0,0 +1,223 @@
+                    WHERE value > ${minValue}
+                    AND value < ${maxValue}
+                    AND name LIKE ${namePattern}
+                    ORDER BY value DESC`
+
+                expect(example.id).to.equal("test2")
+                expect(example.value).to.equal(20)
+            }),
+        ))
+
+    it("should handle SQL tag parameters with NULL values", () =>
+        Promise.all(
+            connections.map(async (connection) => {
+                const repo = connection.getRepository(PostgresExample)
+
+                await repo.save([
+                    { id: "null1", value: null },
+                    { id: "null2", value: 10 },
+                ])
+
+                const examples = await connection.sql`
+                    SELECT * FROM example WHERE value IS ${null}
+                `
+
+                const ids = examples.map((e: PostgresExample) => e.id)
+
+                expect(examples).to.have.length(1)
+                expect(ids).to.have.members(["null1"])
+            }),
+        ))
+
+    it("should handle SQL tag parameters with boolean values", () =>
+        Promise.all(
+            connections.map(async (connection) => {
+                const repo = connection.getRepository(PostgresExample)
+
+                await repo.save([
+                    { id: "true1", active: true },
+                    { id: "false1", active: false },
+                ])
+
+                const examples = await connection.sql`
+                    SELECT * FROM example WHERE active = ${true}
+                `
+
+                const ids = examples.map((e: PostgresExample) => e.id)
+
+                expect(examples).to.have.length(1)
+                expect(ids).to.have.members(["true1"])
+            }),
+        ))
+
+    it("should handle SQL tag parameters with complex array operations", () =>
+        Promise.all(
+            connections.map(async (connection) => {
+                const repo = connection.getRepository(PostgresExample)
+
+                await repo.save([
+                    { id: "array1", tags: ["tag1", "tag2"] },
+                    { id: "array2", tags: ["tag3", "tag4"] },
+                    { id: "array3", tags: ["tag5", "tag6"] },
+                ])
+
+                const searchTags = ["tag1", "tag3"]
+
+                const examples = await connection.sql`
+                    SELECT * FROM example
+                    WHERE EXISTS (
+                        SELECT 1 FROM jsonb_array_elements_text(tags) AS tag
+                        WHERE tag = ANY(${searchTags})
+                    )
+                `
+
+                const ids = examples.map((e: PostgresExample) => e.id)
+
+                expect(examples).to.have.length(2)
+                expect(ids).to.have.members(["array1", "array2"])
+            }),
+        ))
+
+    it("should throw an error when exceeding the maximum number of parameters", async () => {
```

Comment:
What is the purpose of this test? 🤔

Reply:
I wrote this test when we initially defaulted to spreading arrays rather than using a single parameter. Note that it's not the case, this test is useless as it doesn't add any value.